src/index.html

```diff
@@ -65,7 +65,7 @@
             <div class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
             <div class="card__code">Product code: 195434</div>
             <div class="card__review-wrapper">
-              <div class="card__stars-container card__stars--4 stars">
+              <div class="card__stars-container stars card__stars--4">
```

Comment:
```suggestion
              <div class="card__stars-container stars stars--4">
```
в такому вигляді буде доречніше, модифікатор саме на блок додаємо, щоб не стилізувати stars--4 в файлі card, або не використовувати card в stars